apps/menu-bar/src/modules/FilePickerModule.ts

```diff
@@ -9,11 +9,8 @@ const FilePickerModule: FilePickerModuleType = NativeModules.FilePicker;
 
 export default {
   ...FilePickerModule,
-  getAppAsync: () => {
-    return FilePickerModule.pickFileWithFilenameExtension(
-      ['apk', 'app', 'gzip', 'ipa', 'tar'],
-      'Select'
-    );
+  getAppAsync: async () => {
+    return await FilePickerModule.pickFileWithFilenameExtension([], 'Select');
```

Comment:
We should specify the extensions that we support

Reply:
On my system, the `.app` extension was grayed out until I removed the file extensions.

Reply:
![Screenshot 2023-10-01 at 1 31 20 PM](https://github.com/expo/orbit/assets/6577821/473709ab-8238-48d1-85d0-e5e430cec275)

Reply:
After my change:

![Screenshot 2023-10-01 at 1 33 23 PM](https://github.com/expo/orbit/assets/6577821/ec3a5ffc-3922-45fc-bd6e-40cd3ea4d20e)

Reply:
Ah I found the fix -- it was an issue in the native macOS module.